Source/CBLCookieStorage.h

```diff
@@ -0,0 +1,61 @@
+//
+//  CBLCookieStorage.h
+//  CouchbaseLite
+//
+//  Created by Pasin Suriyentrakorn on 2/18/15.
+//  Copyright (c) 2015 Couchbase, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class CBLDatabase;
+
+/** NSNotification posted when the cookies stored in the CBLCookieStorage instance have changed. 
+    The notification does not contain a userInfo dictionary */
+extern NSString* const CBLCookieStorageCookiesChangedNotification;
+
+/** NSNotification posted when the acceptance policy of the CBLCookieStorage instance has changed. 
+    The notification does not contain a userInfo dictionary. */
+extern NSString* const CBLCookieStorageAcceptPolicyChangedNotification;
```

Comment:
Is this needed for anything? I couldn't find any code in the PR that observes it.